packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.types.ts

```diff
@@ -173,66 +187,75 @@ export interface IDatePickerProps extends React.Props<DatePicker> {
 
 export interface IDatePickerStrings {
   /**
-   * An array of strings for the full names of months.
-   * The array is 0-based, so months[0] should be the full name of January.
+   * Error message to render for TextField if isRequired validation fails.
    */
-  months: string[];
+  isRequiredErrorMessage?: string;
 
   /**
-   * An array of strings for the short names of months.
-   * The array is 0-based, so shortMonths[0] should be the short name of January.
+   * Error message to render for TextField if input date string parsing fails.
    */
-  shortMonths: string[];
+  invalidInputErrorMessage?: string;
 
   /**
-   * An array of strings for the full names of days of the week.
-   * The array is 0-based, so days[0] should be the full name of Sunday.
+   * Error message to render for TextField if date boundary (minDate, maxDate) validation fails.
    */
-  days: string[];
+  isOutOfBoundsErrorMessage?: string;
 
   /**
-   * An array of strings for the initials of the days of the week.
-   * The array is 0-based, so days[0] should be the initial of Sunday.
+   * An array of strings for the full names of months.
+   * The array is 0-based, so months[0] should be the full name of January.
+   * @deprecated Use 'calendarProps' with 'strings: ICalendarStrings' instead
    */
-  shortDays: string[];
+  months?: string[];
```

Comment:
Instead of deprecating, it seems better to just make IDatePickerStrings extend ICalendarStrings, this way you don't have to change the existing API. You should make sure that the interface to existing consumer will not break also.